.github/workflows/ci.yml

```diff
@@ -249,6 +249,34 @@ jobs:
         with:
           files: ./coverage.txt
 
+  vendor:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version-file: 'go.mod'
+          check-latest: true
+          cache: true
+      -
+        name: Build vendor
+        run: |
+          go mod tidy
```

Comment:
`tidy` is not relevant here, as we enforce go.sum to be up-to-date as part of the PR workflow

Reply:
Fixed